### PR TITLE
Feat : Added categories page

### LINKED ISSUE #8: Update: category.html
Added links of rules page and question page in category.html

### PR DESCRIPTION
**Pull request by RahulSingh9131**
1. categories page
CategoriesPage

### CODE REVIEW

File: css/screens/categories.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+}
+.container{
+    max-width: 81rem;
+    width: 100%;
+    margin: 0 auto;
+    padding: 0 3rem;
+}
+header .container{
+    position: relative;
+    z-index: 10;
```
Comment (srejitk) [inline css/screens/categories.css]: Any particular reason for keeping a high Z - Index? Any other elements conflicting? 

If not I suggest keeping it in the single digits.

**Review by srejitk — APPROVED** [review]
Just a small suggestion mentioned below. Also, add comments please 🥲

File: css/screens/categories.css (hunk 1 above)
Comment (RahulSingh9131) [reply css/screens/categories.css]: sure will remove that

File: css/screens/categories.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    display: inline-block;
+}
+.section .btn{
+    margin-top: var(--mb-1);
+}
+
+
+
+
+
```
Comment (rahulcs754) [inline css/screens/categories.css]: remove space

**Review by rahulcs754 — APPROVED** [review]
I liked your category page 
remove space

File: css/screens/categories.css (hunk 2 above)
Comment (RahulSingh9131) [reply css/screens/categories.css]: sure will remove that

**Pull request merged**